src/box/lua/config/applier/credentials.lua

```diff
@@ -617,7 +616,7 @@ local function sync_privileges(credentials, obj_to_sync)
     -- revoked manually (e.g. with `box.schema.{user,role}.revoke()`).
     -- However, defaults should never be revoked, so target state B is enriched
     -- with them before step 3.
-    local function sync(role_or_user, name, config_privileges)
+    local function sync(role_or_user, name, privileges)
```

Comment:
I am not sure about this change. Originally `config_` was a way to distinguish data format, that could be config, box and IR. Maybe change `config` to `source` everywhere instead of just removing `config_`?